test/net/sourceforge/kolmafia/textui/ParserTest.java

```diff
@@ -1333,6 +1373,20 @@ public void testScriptValidity(ScriptData script) {
 
     // This will fail if an exception is thrown.
     script.parser.parse();
-    assertEquals(((ValidScriptData) script).tokens, script.parser.getTokensContent(), script.desc);
+    assertEquals(((ValidScriptData) script).tokens, getTokensContent(script.parser), script.desc);
+    if (((ValidScriptData) script).positions != null) {
+      assertEquals(
+          ((ValidScriptData) script).positions, getTokensPosition(script.parser), script.desc);
+    }
+  }
+
+  public static List<String> getTokensContent(final Parser parser) {
```

Comment:
I would reduce visibility of these methods, probably to private unless you anticipate some other file calling them...?

(Also a very minor nit: I'd probably call these "getTokenContents", "getTokenPositions", but I recognize it's patterened on the existing function.)

Reply:
well, there's technically an apostrophe after the s:
"get tokens' content"...

Reply:
Well, you can either read it that way, or you can treat token as an adjective. "get tokens' contents" just doesn't roll off the tongue, and the nit is more that something like "get tokens' position" feels misleading given that they don't share a single position.